File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task1_CLAUDE_claude-3-5-sonnet-20240620.java

import java.util.ArrayList;
import java.util.List;

public class BusPassengers {
    public static int countPassengers(ArrayList<int[]> stops) {
        int passengers = 0;
        for (int[] stop : stops) {
            passengers += stop[0] - stop[1];
        }
        return passengers;
    }

    public static void main(String[] args) {
        ArrayList<int[]> stops = new ArrayList<int[]>();
        stops.add(new int[]{10, 0});
        stops.add(new int[]{3, 5});
        stops.add(new int[]{2, 5});
        System.out.println(countPassengers(stops));  // Output: 5
    }
}

File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task1_CLAUDE_claude-3-5-sonnet-20240620.cpp

#include <iostream>
#include <vector>

int countPassengers(std::vector<std::pair<int, int>>& stops) {
    int passengers = 0;
    for (const auto& stop : stops) {
        passengers += stop.first - stop.second;
    }
    return passengers;
}

int main() {
    std::vector<std::pair<int, int>> stops = {{10, 0}, {3, 5}, {2, 5}};
    std::cout << countPassengers(stops) << std::endl;  // Output: 5
    return 0;
}
